code/modules/modular_computers/file_system/programs/generic/deck_management.dm

```diff
@@ -7,6 +7,7 @@
 	filename = "deckmngr"
 	filedesc = "Deck Management"
 	nanomodule_path = /datum/nano_module/deck_management
+	required_access = list(access_cargo, access_bridge)
```

Comment:
Note that this is expected to be AND; use list(list()) for OR (like the original).